test/viewmodels/CashAccountV2ViewModelSpec.scala

```diff
@@ -44,18 +39,32 @@ class CashAccountV2ViewModelSpec extends SpecBase {
 
   "apply method" should {
 
-    "return correct contents" in new Setup {
+    "return correct contents when maxTransactionsExceeded is None" in new Setup {
```

Comment:
Make use of when instead of copying in the description . It could be done like below

"return correct contents" when {
  "maxTransactionsExceeded is None" in {
   }

"maxTransactionsExceeded is true" in {
 }
}